app/fruits/[fruitName]/FruitCommentForm.js

```diff
@@ -0,0 +1,29 @@
+'use client';
+
+import { useRouter } from 'next/navigation';
+import { useState } from 'react';
+import { createComment } from './actions';
+
+export default function FruitCommentForm(props) {
+  const [comment, setComment] = useState(props.fruitComment);
+  const router = useRouter();
+
+  return (
+    <form>
+      <textarea
+        value={comment}
+        onChange={(event) => {
+          setComment(event.currentTarget.value);
+        }}
+      />
+      <button
+        formAction={async () => {
+          router.refresh();
```

Comment:
let's add back the comment about the workaround, was lost somewhere in the last commits

Reply:
2900364